support/langkit_support.gpr

```diff
@@ -12,6 +12,14 @@ library project Langkit_Support is
    Library_Kind_Param : Library_Kind_Type := external
      ("LIBRARY_TYPE", external ("LANGKIT_SUPPORT_LIBRARY_TYPE", "static"));
 
+   type OS_Kind is ("windows", "unix", "osx");
+   OS : OS_Kind := External ("LANGKIT_SUPPORT_OS", "windows");
```

Comment:
We would like to keep things working with default options: please replace `"windows"` with `""` here and adjust the rest of the patch to avoid defining `Library_Version` in that case?